### ISSUE TITLE
eodoc: generator of HTML documentation from EO sources

### ISSUE THREAD

**Issue by yegor256** [objectionary/eoc#369]
We need to create a documentation generator. It has to be called as `eoc doc`. It should execute `register` and `parse` beforehand, and then simply scan all available `.xml` files take object documentation from it, for example:

```
# This is the documentation, explaining how to use
# this object:
#
# ```
# this is fixed-font
# Markdown block
# ```
#
# Markdown can be here [too](some-link)
[x] > foo
  Q.io.stdout > @
    "Hello, world!\n"
```

Here, the entire block above the object will be in the XML produced by the parser. We should create `xxx.html` for each `xxx.xml` file. It would be great to use XSL for this generator. Also, we should create HTML files for each package, similar to how JavaDoc does it.

**Comment by Thayorns** [comment]
@yegor256, just wanted to clarify - could it be that this issue is outdated?
I found this already existing command - `docs` - and it seems to do the desired logic with `XMIR` to `html`

<img width="691" height="529" alt="Image" src="https://github.com/user-attachments/assets/d5430443-fc9a-4361-9311-3359118d793d" />

Forgive me if I'm wrong

**Comment by yegor256** [comment]
@Thayorns indeed, the skeleton is there, but it generates empty HTML files. That's why the issue is still open.
